src/presenter/point-presenter.js

```diff
@@ -97,13 +95,25 @@ export default class PointPresenter{
     }
   };
 
+  escKeyDownHandler = (evt) => {
+    if(evt.key === 'Escape'){
+      evt.preventDefault();
+      this.closeEditMode();
+      document.removeEventListener('keydown', this.escKeyDownHandler);
+    }
+  };
+
   changeFavorite = () => {
     this.#handleDataChange({
       ...this.#pointData,
       isFavorite: !this.#pointData.isFavorite
     });
   };
 
+  destroy(){
+    remove(this.#view);
```

Comment:
Если точка открыта, она не удалится
```suggestion
    remove(this.#pointEditComponent);
    remove(this.#pointViewComponent);
```